### PR TITLE
Add missing Spanish translations for keycloak.v2 admin theme

### LINKED ISSUE #25041: Incomplete Spanish translations for Admin UI
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

translations

### Describe the bug

There are many i18n keys that remain untranslated for Spanish in the Admin UI:
image

Compare the number of keys in:
https://github.com/keycloak/keycloak/blob/1d4ce3d8e9dbc9af996f9248996c78e145902a35/js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_en.properties#L1-L2920
vs:
https://github.com/keycloak/keycloak/blob/1d4ce3d8e9dbc9af996f9248996c78e145902a35/js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_es.properties#L1-L273

### Version

23

### Expected behavior

All Admin UI i18n keys should be translated for Spanish

### Actual behavior

Many keys remain untranslated

### How to Reproduce?

1. Enable **localization** in **Realm Settings**
2. Add `español` to the list of **Supported locales**
3. Logout
4. Login with Locale set to `Español`

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by daniel-shuy**
Fixes #25041

Credits to @Stephen1234567 for the translations

### CODE REVIEW

**Comment by jonkoops** [comment]
@miquelsi @rmartinc @Aboullos can I ask you to put in a review for this PR?

File: js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_es.properties [hunk 1]
```diff
@@ -1,273 +1,2919 @@
...
-validateSignatureHelp=Habilitar/deshabilitar la validación de firmas de proveedores de identidad (IDP) externos
-clientSignatureHelp=¿Firmará el cliente sus peticiones y respuestas SAML? ¿Y deberían ser validadas?
-title=Sesiones
-keyPasswordHelp=Contraseña para acceder a la clave privada contenida en el archivo
-verifyEmail=Verificar email
-frontchannelLogout=Desonexión en primer plano (Front Channel)
-formatOption=Formato
-loginTheme=Tema de inicio de sesión
+claimFilterNameHelp=Nombre de la reclamaci�n esencial
+revocationDescription=Esta es una forma de revocar todas las sesiones activas y tokens de acceso. 'Not before' significa que puedes revocar cualquier token emitido antes de la fecha.
```
Comment (Aboullos) [inline js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_es.properties:112]:
```suggestion
revocationDescription=Esta es una forma de revocar todas las sesiones activas y tokens de acceso. 'No antes' significa que puedes revocar cualquier token emitido antes de la fecha.
```

File: js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_es.properties [hunk 2]
```diff
@@ -1,273 +1,2919 @@
...
+userRegistrationHelpText=Habilitar/deshabilitar la p�gina de registro. Un enlace para el registro tambi�n aparecer� en la p�gina de inicio de sesi�n.
+activeHelp=Establece si las llaves se pueden utilizar para firmar
+addMapperExplain=Si deseas un control m�s detallado, puedes crear un mapeador de protocolo en este cliente
+realmRoles=Roles de reino
+fineGrainOpenIdConnectConfigurationHelp=Esta secci�n se utiliza para configurar ajustes avanzados de este cliente relacionados con el protocolo OpenID Connect.
+searchForUserDescription=Este reino puede tener un proveedor federado. Ver todos los usuarios puede hacer que el sistema se vuelva lento, pero se puede hacer buscando "*". Por favor, busca un usuario arriba.
+expirationHelp=Establece la expiraci�n para eventos. Los eventos caducados se eliminan peri�dicamente de la base de datos.
+webAuthnPolicySignatureAlgorithmsHelp=Qu� algoritmos de firma se deben usar para la Asertaci�n de Autenticaci�n.
+setToNowError=�Error\! No se pudo establecer 'notBefore' en la fecha y hora actuales.
```
Comment (Aboullos) [inline js/apps/admin-ui/maven-resources/theme/keycloak.v2/admin/messages/messages_es.properties:1201]:
```suggestion
setToNowError=�Error\! No se pudo establecer 'No antes' en la fecha y hora actuales.
```

**Review by Aboullos — CHANGES_REQUESTED** [review]
The file is really big but it looks generally good, some suggestions

**Comment by jonkoops** [comment]
@daniel-shuy could you process the review comments?

**Comment by daniel-shuy** [comment]
@Aboullos Huge thanks for reviewing the translations!

@jonkoops Sorry for the delay, I've applied the suggestions

**Comment by jonkoops** [comment]
@Aboullos coud you give this another pass? I think we can merge this pretty soon.

**Comment by Aboullos** [comment]
LGTM!

**Review by Aboullos — APPROVED** [review]

**Review by miquelsi — APPROVED** [review]

**Comment by daniel-shuy** [comment]
Happy New Year! Any chance this could be merged soon?

**Review by jonkoops — APPROVED** [review]

**Comment by jonkoops** [comment]
Approving and merging this in based on previous approval.

**Comment by daniel-shuy** [comment]
Any idea when we can expect this to be released?

**Comment by jonkoops** [comment]
This will be in the next major version, so 24. This should be released in a couple of weeks, but no guarantee.

**Comment by daniel-shuy** [comment]
@jonkoops okay, thanks

**Pull request merged**